### ISSUE TITLE
NVDA Does Not Announce Custom Punctuation With Left and Right Arrow

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#6770]
If you customize a punctuation symbol in NVDA, the screen reader will announce the new symbol name if you're reading by line or sentence, but it will not say the new symbol name if you press LEFT or RIGHT ARROW over the symbol.

For example, if I change "tick" to "apostrophe" in the Punctuation / Symbols dialog, and I LEFT or RIGHT ARROW over an apostrophe character, NVDA reports "tick."

Similarly, if I change "semi" to "semicolon", and I do a Say All, NVDA will say something like "This is a test semicolon I am testing the use of the semicolon character." However, if I LEFT or RIGHT ARROW over the semicolon, NVDA reports "semi."

**Comment by feerrenrut** [comment]
I am not able to reproduce this with NVDA version `next-13821,09325856`.
I have tried reproducing in WordPad, and Chrome text edit fields.

@munawarb Could you please let me know what version of NVDA you are using and what application you were testing in?

**Comment by munawarb** [comment]
Turns out this was user error; there are different ways to treat the punctuation mark depending on context. I didn't notice this before.
